app/uk/gov/hmrc/transitmovementsrouter/models/AuditType.scala

```diff
@@ -0,0 +1,86 @@
+  final case object InvalidationDecision                               extends AuditType("InvalidationDecision")
+  final case object DeclarationAmendment                               extends AuditType("DeclarationAmendment")
+  final case object DeclarationInvalidationRequest                     extends AuditType("DeclarationInvalidationRequest")
+  final case object DeclarationData                                    extends AuditType("DeclarationData")
+  final case object Discrepancies                                      extends AuditType("Discrepancies")
+  final case object GoodsReleaseNotification                           extends AuditType("GoodsReleaseNotification")
+  final case object MRNAllocated                                       extends AuditType("MRNAllocated")
+  final case object ReleaseForTransit                                  extends AuditType("ReleaseForTransit")
+  final case object RecoveryNotification                               extends AuditType("RecoveryNotification")
+  final case object UnloadingPermission                                extends AuditType("UnloadingPermission")
+  final case object UnloadingRemarks                                   extends AuditType("UnloadingRemarks")
+  final case object WriteOffNotification                               extends AuditType("WriteOffNotification")
+  final case object NoReleaseForTransit                                extends AuditType("NoReleaseForTransit")
+  final case object RequestOfRelease                                   extends AuditType("RequestOfRelease")
+  final case object GuaranteeNotValid                                  extends AuditType("GuaranteeNotValid")
+  final case object RejectionFromOfficeOfDeparture                     extends AuditType("RejectionFromOfficeOfDeparture")
+  final case object RejectionFromOfficeOfDestination                   extends AuditType("RejectionFromOfficeOfDestination")
+  final case object ControlDecisionNotification                        extends AuditType("ControlDecisionNotification")
+  final case object RequestOnNonArrivedMovement                        extends AuditType("RequestOnNonArrivedMovement")
+  final case object InformationAboutNonArrivedMovement                 extends AuditType("InformationAboutNonArrivedMovement")
+  final case object PresentationNotificationForThePreLodgedDeclaration extends AuditType("PresentationNotificationForThePreLodgedDeclaration")
+  final case object FunctionalNack                                     extends AuditType("FunctionalNack")
+  final case object PositiveAcknowledge                                extends AuditType("PositiveAcknowledge")
+  final case object ArrivalNotification                                extends AuditType("ArrivalNotification")
+  final case object ForwardedIncidentNotificationToED                  extends AuditType("ForwardedIncidentNotificationToED")
```

Comment:
Only some of these will be important for this service, so we shouldn't include those that are actually for the API layer. Equally, if the API layer has some of these that should only be here on the router, we should make sure we remove them from there.

Reply:
Apart from going through the process flows, is this info to hand in any of our analysis docs?

Reply:
Yes, that big Confluence doc that I wrote to pull everything together and got everyone to review.

Reply:
updated the auditType list